src/server/project.ts

```diff
@@ -1114,6 +1127,11 @@ namespace ts.server {
                 exclude: []
             };
         }
+
+        /*@internal*/
+        protected getProjectRootPath(): Path | undefined {
+            return this.projectRootPath as Path;
+        }

```

Comment:
thiws method isnt needed... There is always projectRootPath set as currentDirectory field on project

Reply:
OK, do you know why `currentDirectory` is not a `Path`?